app/page/info.tsx

```diff
@@ -5,121 +5,113 @@ import { Link } from "wouter"
 import { useProviderContext } from "~app/context/provider"
 import { NavbarLayout } from "~app/layout/navbar"
 import { Path } from "~app/path"
-import { useAccount } from "~app/storage"
-import { format } from "~packages/util/json"
+import { useAccount, useUserOperationLogs } from "~app/storage"
+import {
+  UserOperationStatus,
+  type UserOperationLog
+} from "~background/storage/local"
+import { UserOperation } from "~packages/bundler"
+import address from "~packages/util/address"
 import type { HexString } from "~typing"
 
+type UserOperationUi = {
+  status: UserOperationStatus
+  hash: HexString
+}
+
 export function Info() {
+  const explorerUrl = "https://jiffyscan.xyz/"
+
   const { provider } = useProviderContext()
+  const userOpLogs = useUserOperationLogs()
   const account = useAccount()
+  const [chainName, setChainName] = useState<string>("")
   const [balance, setBalance] = useState<bigint>(0n)
-  const [transactionHashes, setTransactionHashes] = useState<HexString[]>([""])
-  const [internalTransactionHashes, setInternalTransactionHashes] = useState<
-    HexString[]
-  >([""])
-  const [explorerUrl, setExplorerUrl] = useState<string>("")
+  const [balanceLoading, setBalanceLoading] = useState<boolean>(false)
+  const [userOpsUi, setUserOpsUi] = useState<UserOperationUi[]>([])
 
   useEffect(() => {
+    // Retrieve chainName when the Info page is opened
     const asyncFn = async () => {
```

Comment:
nit: We can give it a meaningful name, for instance here, `getChainName`. (Same for other async functions in `useEffect`)